api/serializers.py

```diff
@@ -0,0 +1,43 @@
+from asyncore import write
+from email.policy import default
+from http.client import FOUND
+import re
+from rest_framework import serializers
+from greedybear.models import IOC
+from greedybear.consts import REGEX_IP, REGEX_DOMAIN
+
+class IOCSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = IOC
+        fields = (
+            'name', 
+            'type', 
+            'first_seen', 
+            'last_seen', 
+            'days_seen',
+            'number_of_days_seen',
+            'times_seen', 
+            'log4j', 
+            'cowrie', 
+            'scanner', 
+            'payload_request', 
+            'related_ioc',
+        )
```

Comment:
don't need to specify these fields explicity, just use `fields = serializers.ALL_FIELDS`.

Reply:
As we don't have to expose `related_urls` in the API response i specified the feild which needs to be sent in order to omit the one which we do not want to send.

Reply:
makes sense (`related_urls` could expose info regarding honeypots). Anyway we could explicitly write this down as a comment so we don't forget

Reply:
> to omit the one which we do not want to send.

You can use the `exclude` attribute for that then.